src/main/kotlin/com/safehill/kclient/models/SHRemoteUser.kt

```diff
@@ -36,6 +44,39 @@ data class SHRemoteUser(
         }
     }
 
+    class MapInResultDeserializer : ResponseDeserializable<Map<String, SHRemoteUser>> {
+        private val mapSerializer = MapSerializer(
+            String.serializer(),
+            SHRemoteUser.serializer()
+        )
+
+        override fun deserialize(content: String): Map<String, SHRemoteUser> {
```

Comment:
Define a typealias for user public identifier = String so we can differentiate between - for instance - asset global identifiers as strings and user public identifiers

Reply:
Are we getting user identifier for keys in map? I though we were getting the hashed phone numbers back as key.

Reply:
Ah, that's right. That's why we need more clarity in the types :)

Reply:
What alias name do you recommend?

Reply:
Same as in the Swift codebase
```
public typealias GlobalIdentifier = String
public typealias LocalIdentifier = String
public typealias UserIdentifier = String
public typealias BearerToken = String
```

HashedPhoneNumber is currently not a type alias but it should be